.github/workflows/rust.yml

```diff
@@ -66,6 +66,22 @@ jobs:
       run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "num-bigint-04"
     - name: Cargo check with bigdecimal-04 feature
       run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "bigdecimal-04"
+
+    # TLS-related feature sets.
+    - name: Cargo check with openssl-x feature
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "openssl-010"
+    - name: Cargo check with rustls-x feature
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "rustls-023"
+    # (openssl-x, rustls-x) is checked in tls.yml.
+
+    # Cloud-related feature sets.
+    - name: Cargo check with unstable-cloud and openssl-x features
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "unstable-cloud" --features "openssl-010"
+    - name: Cargo check with unstable-cloud and rustls-x features
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "unstable-cloud" --features "rustls-023" 
+    - name: Cargo check with unstable-cloud, openssl-x and rustls-x features
+      run: cargo check --all-targets --manifest-path "scylla/Cargo.toml" --features "unstable-cloud" --features "openssl-010" --features "rustls-023" 
+
```

Comment:
This is terrible, the static checks are already long :(

Reply:
Any better ideas?

Reply:
Make the features independent (I think this is only problem with backend for cloud, right?), so that they are just adding new enum variants and implementations.
Then just test with no TLS features, and all of them.